### ISSUE TITLE
Code readability improve

### ISSUE THREAD

**Issue by maucon** [TheSwerik/CalcPrimesMultiThread#7]
CalcPrimesMultiThread/Backend/src/Prime/Thread/ThreadMaster.cs 
Line 17 & 18
```
_threadCount = threadCount < 1 ? Environment.ProcessorCount : threadCount;
_threadCount = threadCount > 64 ? 64 : threadCount;
```
can be replaced with:
`Math.Min()`
`Math.Max()`

I think that makes the code overall cleaner and improves the readability.
Should be replaced in all files when usable.

**Comment by maucon** [comment]
Another example:

CalcPrimesMultiThread/Backend/src/Prime/util/CustomConsole.cs
Starting at line 35
```
var index = sb.ToString().LastIndexOf(Environment.NewLine, StringComparison.Ordinal) >= 0
                            ? sb.ToString().LastIndexOf(Environment.NewLine, StringComparison.Ordinal)
                            : sb.Length;
```
```
index = sb.ToString().LastIndexOf(Environment.NewLine, StringComparison.Ordinal) >= 0
                        ? sb.ToString().LastIndexOf(Environment.NewLine, StringComparison.Ordinal) +
                          Environment.NewLine.Length
                        : 0;
```

**Comment by TheSwerik** [comment]
> CalcPrimesMultiThread/Backend/src/Prime/Thread/ThreadMaster.cs
> Line 17 & 18
> 
> ```
> _threadCount = threadCount < 1 ? Environment.ProcessorCount : threadCount;
> _threadCount = threadCount > 64 ? 64 : threadCount;
> ```
> 
> can be replaced with:
> `Math.Min()`
> `Math.Max()`
> 
> I think that makes the code overall cleaner and improves the readability.
> Should be replaced in all files when usable.

First one doesnt work: Either threadcount or, if <1, Processorcount. If I take Math.Max then threadcount == 1 would'nt be possible anymore for example

**Comment by TheSwerik** [comment]
> Another example:
> 
> CalcPrimesMultiThread/Backend/src/Prime/util/CustomConsole.cs
> Starting at line 35
> 
> ```
> var index = sb.ToString().LastIndexOf(Environment.NewLine, StringComparison.Ordinal) >= 0
>                             ? sb.ToString().LastIndexOf(Environment.NewLine, StringComparison.Ordinal)
>                             : sb.Length;
> ```
> 
> ```
> index = sb.ToString().LastIndexOf(Environment.NewLine, StringComparison.Ordinal) >= 0
>                         ? sb.ToString().LastIndexOf(Environment.NewLine, StringComparison.Ordinal) +
>                           Environment.NewLine.Length
>                         : 0;
> ```

There aswell, how? I want to take the index, if it's not -1, if it is -1, take the length.
How would you fix that with Min or Max?

**Comment by TheSwerik** [comment]
but one of them is fixed in 12c1976

**Comment by maucon** [comment]
CalcPrimesMultiThread/Frontend/src/MainWindow.xaml.cs
Starting at line 164
```
if (input > 64) ThreadBox.Text = "64";
else if (input < 1) ThreadBox.Text = "1";
```

**Comment by maucon** [comment]
CalcPrimesMultiThread/Frontend/src/Starter.cs 
Line 27
`if (ShouldOverride) TaskMaster.StartSieve(max <= MaxSieveValue ? max : MaxSieveValue, token);`

**Comment by TheSwerik** [comment]
fixed in 95944b0
